tests/tools/test_import_without_warnings.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from pydantic.warnings import PydanticDeprecatedSince20
+
+
+@pytest.mark.filterwarnings("error", category=PydanticDeprecatedSince20)
+def test_import_tools_without_pydantic_deprecation_warnings():
+    # This test is to ensure that the import of crewai_tools does not raise any Pydantic deprecation warnings.
+    import crewai_tools
+
+    assert crewai_tools
```

Comment:
Should we use [assertLogs](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLogs) or something like that?

Reply:
Warnings are separated from logs.
I've used the [@pytest.mark.filterwarnings](https://docs.pytest.org/en/stable/how-to/capture-warnings.html#pytest-mark-filterwarnings) to convert the warning into an error, this way is easier to identify the root cause because it shows the stack trace.
I'm asserting the import just to use the variable 😆